README.md

```diff
@@ -0,0 +1,65 @@
+# Uyuni RFCs
+
+Many changes, including bug fixes and documentation improvements, can be implemented and reviewed via the normal GitHub pull request workflow. Some changes though are "substantial" - those should be put through an appropriate design process and produce a consensus among the Uyuni core team.
+
+The "RFC" (request for comments) process is intended to provide a consistent and controlled path for new features to enter the project. This process will be adjusted over time as more features are implemented and the community settles on specific approaches to feature development.
+
+## TL;DR
+
+* Fork https://github.com/uyuni-project/uyuni-rfc
+* Copy `00000-template.md` to `active/0000-my-feature.md`
+  - 'my-feature' should be as descriptive as possible.
+  - please don't assign an RFC number yet.
+* Write the content, based on the [RFC template](0000-template.md).
+* Open a Pull Request and discuss it.
+* When the discussion is finished and the PR ready to be merged, lookup the next free number in master and rename the file.
+
+Once the PR is merged, [implementation can start](#first-principles).
```

Comment:
```suggestion
Once the PR is merged, implementation can start.
```

Reply:
Why remove this link?

Reply:
It points nowhere AFAICS. Am I missing anything?

Reply:
It points to the "First principles" heading/anchor below. The link should work.

Reply:
Strange, for some reason it didn't at least in the GitHub preview